example/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "example",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "parcel index.html",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {
+    "react-app-polyfill": "^1.0.0"
```

Comment:
Why is this not a dev dependency?

Reply:
It can be moved to Dev, however this library it's useful to support IE browser, that might not be the case for dev.

Reply:
No problem, lets keep it!